File: pinyougou-manager-web/src/main/webapp/js/service/specificationService.js
app.service('specificationService',function ($http) {
    this.dele = function (selectids) {
        return $http.post('../specification/dele.do',selectids);
    }
    this.search = function (searchEntity,page,rows) {
        return $http.post('../specification/search.do?rows='+rows+"&page="+page,searchEntity);
    }
    this.save = function (entity) {
        return $http.post('../specification/save.do',entity);
    }
    this.findOne = function (id) {
        return $http.get("../specification/findOne.do?id="+id);
    }
    this.getSpecList=function () {
        return $http.get('../specification/getSpecList.do');
    }

});

File: pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/SpecificationService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbSpecification;
import com.pinyougou.pojogroup.Specification;
import entity.PageResult;

import java.util.List;
import java.util.Map;

public interface SpecificationService {
    PageResult search(TbSpecification tbSpecification, int page, int rows);

    void dele(Long[] ids);

    void save(Specification specification);

    Specification findOne(Long id);

    List<Map> getSpecList();

}
